examples/listeners.html

```diff
@@ -62,10 +62,136 @@
       L.DomEvent.on(img._image, 'load', function() {
         var img = this
         img.on('edit', function() {alert('edited')}, img);
-        L.DomEvent.on(img._image, 'mouseup', function() {alert('edited')}, img);
-        L.DomEvent.on(img._image, 'touchend', function() {alert('edited')}, img);
+        // L.DomEvent.on(img._image, 'mouseup', function() {alert('edited')}, img);
+        // L.DomEvent.on(img._image, 'touchend', function() {alert('edited')}, img);
+      // }, img);
+
+      var imagePos = img._image.getBoundingClientRect();
+      var centerX = imagePos.left + imagePos.width / 2;
+      var centerY = imagePos.top + imagePos.height / 2;
+      var centerRadius = imagePos.width / 10;
+      var touchInfo = { alpha: 0, radius: 0, down: false };
+
+      var spinAlpha = 0;
+      var spinSpeed = 0;
+
+      var touchSpeed = 0;
+      var lastTouchAlpha = 0;
+
+      var imgState = { imgReady: false }
+
+      function onTouchStart(e) {
+        if (imgState.imgReady) {
+          return;
+        }
+
+        var {x, y} = getXYFromTouchOrPointer(e);
+        onTouchMove(e);
+        touchInfo.down = true;
+        touchInfo.radius = Math.sqrt(Math.pow(x, 2) + Math.pow(y, 2));
+        lastTouchAlpha = touchInfo.alpha;
+
+      }
+
+      function getXYFromTouchOrPointer(e) {
+        window.e = e;
+        var x = e.touches ? e.touches[0].clientX : e.clientX;
+        var y = e.touches ? e.touches[0].clientY : e.clientY;
+        // alert(x);
+        return { x: x - centerX, y: y - centerY };
+      }
+
+      function onTouchMove(e) {
+        if (imgState.imgReady) {
+          return;
+        }
+        
+        img.editing._deselect();
+        img.editing.dragging.disable();
+        map.dragging.disable();
+        // alert(e);
+        var { x, y } = getXYFromTouchOrPointer(e);
+        touchInfo.alpha = Math.atan2(x, y);
+        e.preventDefault();
+      }
+
+      function onTouchEnd() {
+        if (imgState.imgReady) {
+          return;
+        }
+
+        touchInfo.down = false;
+        spinAlpha = 0;
+        img.editing._select();
+        
+        img.fire("update");
+        // alert("touchend");
+      }
+
+      function tick() {
+        requestAnimationFrame(function() {
+          if (touchInfo.down) {
+            if (touchInfo.radius > centerRadius) {
+              var touchSpeed = touchInfo.alpha - lastTouchAlpha;
+              if (touchSpeed < -Math.PI) { touchSpeed += 2 * Math.PI; }
+              if (touchSpeed > Math.PI) { touchSpeed -= 2 * Math.PI; }
+
+              var spinSpeed = touchSpeed;
+              lastTouchAlpha = touchInfo.alpha;
+            }
+          } else if (touchSpeed) {
+            spinSpeed = touchSpeed * touchInfo.radius / centerRadius;
```

Comment:
So here, I guess I'd prefer we don't give rotation "momentum", you know? What about just using the angle to manipulate the images using the same method as in `RotateScaleHandle`?:

https://github.com/publiclab/Leaflet.DistortableImage/blob/70b22a2ff4e5b7f0c939ea70340746bbade161cc/src/edit/RotateScaleHandle.js#L17-L20

Reply:
@jywarren agreed! I need to go back and debug since I started tracking rotation. I don't think the speed code should actually be doing anything at this point just needs to be removed. The bug I was concerned about is that as you rotate you hit a sort of "bump" and the image stops and starts rotating in the other direction. 

I also implemented rotation via  `Hammer.js` and it was functionally the seem. Going to push that here now

Reply:
Ok Hammer.js implementation is under `index.html`. I'll go over these again this weekend and rebase!

Reply:
Also just realized the `listeners.html` code here wasn't up to date before - Sorry I might have lost track of my branches!

Reply:
also just remembered - our `_rotateBy` takes in a deltaAngle that is tiny because drag is fired so often. This doesn't play nice with touch, so this is why I needed to track rotation. That part is still a little buggy too but works well enough